Leibit.Core/Data/sued/MLI.xml

```diff
@@ -0,0 +1,221 @@
+    </track>
+    <track name="6">
+      <block name="62B6"/>
+    </track>
+  </station>
+
+  <station name="Röthenbach" short="MRTA" refNr="63" scheduleFile="g63_____.abf" localOrderFile="bf63____.abf">
+    <track name="1">
+      <block name="63G1"/>
+      <alternative track="2"/>
+      <alternative track="3"/>
+    </track>
+    <track name="2">
+      <block name="63G2"/>
+    </track>
+    <track name="3">
+      <block name="63G3"/>
+    </track>
+    <track name="51">
+      <block name="63G51"/>
+      <alternative track="2"/>
+      <alternative track="3"/>
+    </track>
+  </station>
+
+  <station name="Hergatz" short="MHGZ" refNr="64" scheduleFile="g64_____.abf" localOrderFile="bf64____.abf">
+    <track name="1">
+      <block name="64G1"/>
+    </track>
+    <track name="2">
+      <block name="64G2"/>
+    </track>
+    <track name="3">
+      <block name="64G3"/>
+    </track>
+  </station>
+
+  <station name="Lindau-Reutin" short="MLIR" refNr="68" scheduleFile="g68_____.abf" localOrderFile="bf68____.abf">
+    <track name="1">
+      <block name="68G1"/>
+    </track>
+    <track name="2">
+      <block name="68G2"/>
+    </track>
+    <track name="3">
+      <block name="68G3"/>
+    </track>
+    <track name="4">
+      <block name="68G4"/>
+    </track>
+    <track name="5">
+      <block name="68G5"/>
+    </track>
+    <track name="6">
+      <block name="68G6"/>
+    </track>
+    <track name="9">
+      <block name="68G9"/>
+    </track>
+  </station>
+
+  <station name="Lindau Hbf" short="MLI" refNr="67" scheduleFile="g67_____.abf" localOrderFile="bf67____.abf">
+    <track name="1">
+      <block name="67G1"/>
+    </track>
+    <track name="2">
+      <block name="67G2"/>
+    </track>
+    <track name="3">
+      <block name="67G3"/>
+    </track>
+    <track name="4">
+      <block name="67G4"/>
+    </track>
+    <track name="5">
+      <block name="67G5"/>
+    </track>
+    <track name="6">
+      <block name="67G6"/>
+    </track>
+    <track name="7">
+      <block name="67G7"/>
+    </track>
+    <track name="8">
+      <block name="67G8"/>
+    </track>
+    <track name="9">
+      <block name="67G9"/>
+    </track>
+  </station>
+
+  <station name="Nonnenhorn" short="MNHN" refNr="73" scheduleFile="g73_____.abf" localOrderFile="bf73____.abf">
+    <track name="1">
+      <block name="73G1"/>
+    </track>
+    <track name="2">
+      <block name="73G2"/>
+    </track>
+  </station>
+
+  <station name="Wasserburg" short="MWBG" refNr="72" scheduleFile="g72_____.abf">
+    <track name="1">
+      <block name="72B1"/>
+    </track>
+  </station>
+
+  <station name="Enzisweiler" short="MENW" refNr="71" scheduleFile="g71_____.abf" localOrderFile="bf71____.abf">
+    <track name="1">
+      <block name="71G1"/>
+    </track>
+    <track name="2">
+      <block name="71G2"/>
+    </track>
+  </station>
+
+  <station name="Lindau-Aeschach" short="MLIA" refNr="66" scheduleFile="g71_____.abf">
```

Comment:
Hier müsste dann aber der Zusatz "Abzw" hin, oder? Also "Abzw Lindau-Aeschach". Es gibt ja Aeschach Hp und Abzw.

Reply:
So wie ich das verstanden habe, wird eine Betriebsstelle immer ohne einen Zusatz genannt. Zumindest in den aktuellen Daten der Bahn ist das der Haltepunkt. Deb Haltepunkt hab ich aber nicht abgebildet, weil er im selben Block liegt, wie eines der Ausfahrgleise des Abzw..
Ich hab mal hier in die Infos der Bahn geschaut: https://geovdbn.deutschebahn.com/isr
Bzw. hier in die Liste der Betriebsstellen: http://www.bahnseite.de/DS100/DS100_main.html
Der Abzw. heißt abweichend zum ESTWSim in echt "MALI", der Hp ist "MLIA"
Soll ich das kürzel entsprechend der aktuellen DDB Info anpassen?

Reply:
Du hast Recht. Dadurch, dass das Ausfahrgleis des Abzweigs der selbe Block ist, in dem der Hp liegt, kriegt man es leider wieder nicht eindeutig hin. Ich glaube, dann würde ich es so lassen. Also den Hp weglassen und den Namen bei "Lindau-Aeschach" belassen.

Was mich mal interessieren würde, wie das echte LeiDis-FI sowas händelt. So wie ich das verstanden habe, werden dort ja auch nur die Daten der Zugnummernmeldeanlage abgegriffen, sodass die Datengrundlage auf der LeiDis-FI arbeitet in etwa dieselbe sein müsste wie in der Simulation. Weißt du zufällig etwas zu dem Thema?

Reply:
Meine FDL Kontakte haben leider keinen Abzweig bei sich im Stellbereich.
Was ich mir als Möglichkeit vorstellen könnte beim HP, je nach Richtung, einen anderen Block angeben.
Ich habe ihn hier jetzt mal weggelassen.